src/preview.js

```diff
@@ -12,6 +12,36 @@ const getPreviewHeader = ( lang, imageUrl = '' ) =>{
 	`.trim()
 	},
 
+	getPreviewBody = ( lang, title, type, isTouch = false ) => {
+		const getMessage = () => {
+				if ( type === 'disambiguation' ) {
```

Comment:
Since we have distinct entry points (renderPreview, renderError, etc) we shouldn't have those conditionals. Each entry point should compose it's own UI based on re-usable elements. In other words, `renderDisambiguation()` is the only one to know or care about its own body content.